tonic/models/vic/vic.py

```diff
@@ -67,8 +67,18 @@ def run(self, global_param, logdir=None):
 
         return self.returncode
 
-    def _call_vic(self, *args):
-        vic_args = [self.executable] + [a for a in args]
+    def _call_vic(self, *args, mproc_num=None):
+
+	vic_args = []
```

Comment:
style comment - you are mixing tabs and spaces.  PEP8 tells us to use spaces (http://legacy.python.org/dev/peps/pep-0008/#tabs-or-spaces).